src/OneExplorer.ts

```diff
@@ -27,6 +28,7 @@ enum NodeType {
 interface Node {
   type: NodeType;
   name: string;
+  path: string;
```

Comment:
`path` also could be get from `uri.fsPath`.
So if `uri.fsPath` retuns same value with `path`, we may not need to introduce `path`.

Reply:
@hyunsik-yoon I'd thought about that, but I introduced the field to make it explicitly mandatory. I assumed that the original code used uri aside from a container of path, but it doesn't. I'd rather remove uri then :-D.

Reply:
`uri` is also used to determine whether this file is a file or directory. And it can be useful to add directory or watch the file changed event. So, I don't want to recommend removing the uri property.

Reply:
While looking at many usage of `uri`,
it looks like vscode handles `uri` instead of file path. 
e.g., `export function showOpenDialog(options?: OpenDialogOptions): Thenable<Uri[] | undefined>;` which is defined in `vscode.window` returns `Uri` and `OpenDialogOptions` also gets `Uri` as default path.

vscode Uri has the following comment and I guess that vscode seems to treat local file and remote file with `Uri`.
```
     A universal resource identifier representing either a file on disk
     or another resource, like untitled resources.
```

At the beginning of ONE-vscode, we  also discussed that user might not want to copy a large model file in a _remote_ file server. Not sure how to handle such case but having Uri could help such thing later, I guess.

Reply:
Sure :-D

Reply:
Well the thing is, it's right that vscode handles uri instead of its file path. But in this module, we only use 'fspath' property.

> uri is also used to determine whether this file is a file or directory. 

It don't even handle 'wehther this file is a file or directory' its own, it uses `fs` module on 'fspath' property. 
In vscode extension examples, it uses string path instead of uri too.

> vscode Uri has the following comment and I guess that vscode seems to treat local file and remote file with Uri.

Here in this module, I don't use `uri`'s properties nor methods. I tried to look into its methods / properties but it has like 'query' / 'fragment' / 'authority' properties which looks like to  be very much usable in a web.

To make long story short, here we handle with only 'path' information and don't use any useful methods in 'uri', so I'd prefer to use string path.

Reply:
Maybe it's that I don't understand how uri is useful here, but I cannot see its usage as of now. May I use string path and maybe change into uri later when the need is there?

Reply:
Okay, I agree with your comments. Let's use file path instead of `vscode.Uri`. I don't know what it will happen when we don't use this, let's think about it later.

Reply:
https://github.com/Samsung/ONE-vscode/pull/543 I implemented Node as a class to resolve this discussion. I think the problem arises as Node manipulates on path more than expected.